python/QC3.py

```diff
@@ -0,0 +1,31 @@
+from subprocess import CalledProcessError
+from wrappers import runCommand
+from options import parser
+
+args = parser.parse_args()
+cmd = ["python","$GEM_BASE/python/Produce_Config_File.py"]
+
+for f in args.file:
+	cmd.append(f)
+	pass
+
+cmd.append("--CanvTitleX=Time #left(s#right)")
+cmd.append("--CanvTitleY=Pressure #left(mbar#right)")
+cmd.append("--SelectColumnX= 1")
+cmd.append("--SelectColumnY= 2")
+cmd.append("--SelectRowStart= 1")
+cmd.append("--SelectRowEnd= 61")
+cmd.append("--CanvRangeX= 0,3600")
+cmd.append("--CanvRangeY= 0,35")
+
+
+if len(args.file)==1:
+	filetype=f[int(f.index('.')):]
+	filename = f
+	filename= filename.replace(filetype,"")
+	cmd.append("--OutputName="+filename)
```

Comment:
Are you sure this doesn't over-write the input file name?  I think this should be made to be more descriptive as requested previously.

e.g. how does the user distinguish between the input file and the produced output config file?  Generally the `.cfg` ending is not enough, consider a better way of doing this.  For example `config_X.cfg` would be better.

Apply this to all other scripts.